frontend/src/components/BlockDetails/BlockDetailsContainer.vue

```diff
@@ -36,15 +36,12 @@ const paginationVars = {
 }
 
 type Props = {
```

Comment:
Somewhere we must be calling `<BlockDetailsContainer :id=... >` which should also change and we need to make sure that the hash is used then.

Reply:
Alternatively, we can replace `id` with `blockHeight` and use that as identity.

Reply:
Why would we want to do that? I do not understand why we would prefer to be using two different identifiers across the same frontend.

Reply:
The `id` and `blockHeight` contain the same value. Just replaceing the name should cause the least change to be done. As a second step, I agree we should either only use the `blockHeight` or only the `blockHash` to uniquely identity a block at the front-end as much as possible.

Reply:
`id` is not the same as block height for the `.NET` backend, hence the bug we are seing.
Plus we should not rely on any internal details of `id` for the rust backend, since the contract between the frontend and API for `id` is 'just some identifier' and not a particular identifier.

Reply:
I will remove it all the way through, however just to warn it is a rather substantial change.

Reply:
But imo also a lot more cleaner